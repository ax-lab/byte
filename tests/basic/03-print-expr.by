let s = 'something'
print 'say ' + s

let a = 'hello'
let b = 'world'
print a + ' ' + b + '!!!'

let x = 1
let y = 2
let z = 3
print 'x = ' + x + ', y = ' + y + ', z = ' + z

let a1 = 10
let a2 = 4
let a3 = 2
let answer = a1 * a2 + a3

print 'The answer to life the universe and everything is ' + answer + '.'
