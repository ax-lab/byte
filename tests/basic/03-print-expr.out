say something
hello world!!!
x = 1, y = 2, z = 3
The answer to life the universe and everything is 42.
